### PR TITLE
SettingsController for Reservations and puzzle

### PR DESCRIPTION
**Pull request by amihaiemil**
PR for #63 

* Moved Reservation Settings methods into a class
* Created SettingsController and left puzzles

### CODE REVIEW

File: timeless/__init__.py [hunk 1]
```diff
@@ -67,7 +67,7 @@ def register_api(app, view, endpoint, url, pk="id", pk_type="int"):
 
 def register_endpoints(app):
     app.register_blueprint(auth_views.bp)
-    app.register_blueprint(reservations_views.bp)
+    #app.register_blueprint(reservations_views.bp)
```
[the remark this answers is not in the document]
Comment (amihaiemil) [reply timeless/__init__.py]: @paulodamaso it's not needed anymore since settings API methods are now in a class (same as Comments)

**Comment by amihaiemil** [comment]
@paulodamaso I'll open a new PR (have to fork the new public repo) with the changes you pointed out :)

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#128](https://github.com/timelesslounge/timelessis/issues/128) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa)

**Comment by ypshenychka** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Order was finished, quality is "good": +20 point(s) just awarded to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request closed without merging**